source/user-manual/capabilities/system-inventory/generating-system-inventory-reports.rst

```diff
@@ -13,11 +13,11 @@ Property-specific report
 
 This feature allows you to export ``CSV`` reports of a specific property of an endpoint. For example, you can generate a report of the installed software on an endpoint. This kind of report is only available for processes and installed software. 
```

Comment:
**suggestion**: The sentence `This kind of report is only available for processes and installed software.` could be removed because the ability to export is in each table of inventory data.